### PR TITLE
SPUI-1153 > Add amplitude

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart [hunk 1]
```diff
@@ -69,11 +69,12 @@ class SendByPhoneConfirmNotifier
 
       if (!mounted) return;
       _updateIsResending(false);
+      // sAnalytics.kycPhoneConfirmed();
```
Comment (eli1stark) [inline lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart:72]: why comment?

File: lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart [hunk 2]
```diff
@@ -69,11 +69,12 @@ class SendByPhoneConfirmNotifier
...
       onSuccess();
     } catch (error) {
       _logger.log(stateFlow, 'transferResend', error);
       _updateIsResending(false);
-
+      // sAnalytics.kycPhoneConfirmFailed('transferResend');
```
Comment (eli1stark) [inline lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart:77]: why comment?

File: lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart:72]: done

File: lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart (hunk 2 above)
Comment (DenisMartych) [reply lib/app/shared/features/send_by_phone/notifier/send_by_phone_confirm_notifier/send_by_phone_confirm_notifier.dart]: done

**Pull request merged**